.github/workflows/update-dispatch.yml

```diff
@@ -28,8 +27,14 @@ jobs:
             skip: false
           - name: ${{ github.ref_name }}
             skip: true
-        update_source:
-          - holochain-nixpkgs
+        update:
+          - source: "versions/0_1"
+            cmd: "nix run --override-input versions ./versions/0_1 .#scripts-repo-flake-update"
```

Comment:
where is this script?

Reply:
this is defined at https://github.com/holochain/holochain/blob/15394725add885b9cc89176b14460f8c1a22fcd6/nix/modules/scripts.nix#L34-L44